plugins-extra/query_enhancements/public/query_assist/components/query_assist_input.tsx

```diff
@@ -20,18 +20,49 @@ export const QueryAssistInput: React.FC<QueryAssistInputProps> = (props) => {
   const [suggestionIndex, setSuggestionIndex] = useState<number | null>(null);
   const [value, setValue] = useState(props.initialValue ?? '');
 
-  const recentSearchSuggestions = useMemo(() => {
+  const sampleDataSuggestions = useMemo(() => {
+    switch (props.selectedIndex) {
+      case 'opensearch_dashboards_sample_data_ecommerce':
+        return [
+          'How many unique customers placed orders this week?',
```

Comment:
A future iteration, if we want to consider making this apart of the registration of sample data. I have the code ready to go: https://github.com/opensearch-project/OpenSearch-Dashboards/blob/feature/discover-next/plugins-extra/query_enhancements/server/plugin.ts#L49